src/twitter/Timeline.js

```diff
@@ -269,18 +275,26 @@ class Timeline {
     return false;
   }
 
-  is_feed_exceeding_user_limit (tweets, isTimeline) {
-    if((isTimeline ? this.config.timelines : this.config.hashtags).single_account_fallback) {
-      return false;
-    }
-
-    const pollInterval = isTimeline ? TIMELINE_POLL_INTERVAL_MS : HASHTAG_POLL_INTERVAL_MS;
-    // Rough time since we last polled this feed
-    const currentPollInterval = (isTimeline ? this._timelines : this._hashtags).length * pollInterval;
-    // Number of new profiles that we can accept in this timespan.
-    const maxNProfiles = Math.ceil((currentPollInterval / 60000) * NEW_PROFILE_THRESHOLD_MIN);
-    const userIds = new Set(tweets.map((tweet) => {tweet.id_str})).size;
-    return userIds > maxNProfiles;
+  roomsExceedingJoinThreshold (tweets, rooms, isTimeline) {
+    if(!(isTimeline ? this.config.timelines : this.config.hashtags).single_account_fallback) {
+      return new Set();
+    }
+    // Find any rooms that exceed the threshold.
+    const result = ([...rooms]).filter((room) => {
+      const currentMembers = this._members_rooms.get(room) || new Set();
```

Comment:
might be slightly nicer to have a constant `EMPTY_SET`, which is set to `new Set()`, as it looks like `currentMembers` isn't modified.